components/org.wso2.carbon.identity.application.authenticator.iwa/src/main/java/org/wso2/carbon/identity/application/authenticator/iwa/IWAFederatedAuthenticator.java

```diff
@@ -112,7 +112,7 @@ protected void processAuthenticationResponse(HttpServletRequest request, HttpSer
 
         if (StringUtils.isEmpty(userStoreDomain)) {
             context.setSubject(
-                    AuthenticatedUser.createFederateAuthenticatedUserFromSubjectIdentifier(authenticatedUserName));
+                    AuthenticatedUser.createFederateAuthenticatedUserFromSubjectIdentifier(fullyQualifiedName));
```

Comment:
Idea behind using the 'authenticatedUserName' (username after removing the Kerberos domain) was to remove the Kerberos Domain info from username in the ticket. Any reason why we should have it in the username?